doc/api/synopsis.md

```diff
@@ -9,9 +9,43 @@ Please see the [Command Line Options][] document for information about
 different options and ways to run scripts with Node.js.
 
 ## Example
-
 An example of a [web server][] written with Node.js which responds with
-`'Hello World'`:
+`'Hello World!'`:
+
+Firstly, make sure you have downloaded Node.js from [Node.js Official website](http://nodejs.org/#download).
+Then, follow this [installation guide](https://nodejs.org/en/download/package-manager/).
+
+
+Now, create an empty project folder called `projects`, navigate into it:
+
+Linux and Mac:
```

Comment:
May be UNIX rather than Linux and Mac ?

Reply:
We usually go with `POSIX` or else `Linux/macOS` I think.

Reply:
```
doc/api> grep -i "UNIX" * | wc -l
      40
```
We seem to be using `UNIX` abundantly to refer UNIX-like systems.

If we use Linux and MAC, it leaves an impression of either:
 - Only Linux and MAC are supported
 - The example holds good only for Linux and MAC

Reply:
@gireeshpunathil I'm convinced. Let's go with `UNIX` (or `Unix` or `UNIX-like`) and consolidating terminology/typography across documents can happen some other time. Certainly "Linux and macOS" would explicitly exclude (for example) AIX which certainly is incorrect in this case.